pantheon-bundle/frontend/src/app/routes.tsx

```diff
@@ -75,7 +76,15 @@ class Routes extends Component<IAppState> {
         exact: false,
         icon: null,
         label: '', // Empty because we are using the Brand component to render the text.
-        path: '/:data',
+        path: '/module/:data',
```

Comment:
So, are we not using module endpoint now onwards, instead it is /module/:data ?

Reply:
please see my response above regarding why i updated the routes.